lambda/templates/ecr_scan_lambda.json.tpl

```diff
@@ -0,0 +1,26 @@
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Effect": "Allow",
+      "Action": [
+        "ecr:StartImageScan",
```

Comment:
Could you order the `Action`s alphabetically?

Reply:
Yep, I've sorted them.